src/components/mols/index.ts

```diff
@@ -7,6 +6,8 @@ export { default as CouponActivatedPage } from "./CouponActived";
 export { default as CouponActived } from "./CouponActived";
 export { default as UserCoupons } from "./UserCoupons";
 export { default as FastLoginPage } from "./FastLoginPage";
-export { default as SiteHeader } from "./SiteHeader";
+export { default as SiteHeader } from "../orgs/SiteHeader";
```

Comment:
Why an **organism** is being export in **mols/index**?

Reply:
i think this brench is not the last one

Reply:
in my brench theres no ../orgs""

Reply:
Why this conversation was resolved? Is this export correct?